marshmallow_jsonapi/validate.py

```diff
@@ -0,0 +1,25 @@
+from marshmallow.validate import Validator, ValidationError
+
+class ForeignKey(Validator):
```

Comment:
That's reasonable; I threw it in cause someone mentioned it, but people will probably want their own validators anyway (e.g. to check the id, whatever type it may be, exists in the records of the related type)